src/apps/ENGINE/ifs.cpp

```diff
@@ -362,38 +360,34 @@ bool IFS::VoidSym(char symbol)
 
 bool IFS::LoadFile(const char *_file_name)
 {
-    uint32_t dwR;
-
     if (_file_name == nullptr)
+    {
         return false;
-    auto *const fh = fs->_CreateFile(_file_name, GENERIC_READ, FILE_SHARE_READ, OPEN_EXISTING);
-    if (fh == INVALID_HANDLE_VALUE)
-        return false;
-
-    const auto file_size = fs->_GetFileSize(fh, nullptr);
-    if (file_size == INVALID_FILE_SIZE)
+    }
+    auto fileS = fs->_CreateFile(_file_name, std::ios::binary | std::ios::in);
+    if (!fileS.is_open())
     {
-        fs->_CloseHandle(fh);
```

Comment:
add some verbose logging?

Reply:
There is file called `project.df` and as I understand it is some debug stuff and it's not necessary to have it. But I got error exception if I'll add
`core.tracelog->trace("Can not load file: %s", _file_name);`
here